targets/ramips-mt7620

```diff
@@ -32,6 +32,18 @@ device('nexx-wt3020-8m', 'wt3020-8M', {
 })
 
 
+-- TP-Link
+
+local tplink_region_suffix = ''
+if (env.GLUON_REGION or '') ~= '' then
+        tplink_region_suffix = '-' .. env.GLUON_REGION
+end
+
+device('tp-link-archer-c50', 'ArcherC50v1', {
+        factory = '-squashfs-factory' .. tplink_region_suffix,
```

Comment:
Same story as for the Archer C2 - does this image actually work somewhere? The OpenWrt wiki only talks about a similar TFTP-concat using the sysupgrade image.

Reply:
If you have initial Release of the Stock firmware it works (tested with tftp recover to stock initial firmware and then flash from web interface)